iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/TieLineImpl.java

```diff
@@ -6,11 +6,11 @@
  */
 package com.powsybl.iidm.network.impl;
 
-import java.util.Objects;
-
 import com.powsybl.iidm.network.TieLine;
 import com.powsybl.iidm.network.ValidationException;
 
+import java.util.Objects;
+
```

Comment:
You should revert this change

Reply:
Sorry, my bad 
Use Intellij configuration "Optimize imports on the fly"